package.json

```diff
@@ -20,5 +20,8 @@
   "workspaces": [
     "apps/*",
     "packages/*"
-  ]
+  ],
+  "dependencies": {
```

Comment:
Dep should be in the apps/frontend folder

Reply:
Moved to Frontend/package.json